src/util/ChartUtils.ts

```diff
@@ -667,7 +667,7 @@ export interface RechartsScale {
 
 export type AxisPropsNeededForTicksGenerator = {
   duplicateDomain?: ReadonlyArray<AxisTick>;
-  realScaleType?: 'scaleBand' | 'band' | 'point' | 'linear';
+  realScaleType?: 'scaleBand' | string;
```

Comment:
?

Reply:
This function accepts all scales, not sure why was it limited to only these four before. All usages were hidden behind `any`.